test/foundry/fixtures/HolographERC721Fixture.t.sol

```diff
@@ -0,0 +1,218 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
```

Comment:
Nice. Thanks for adding this fixture. I think it'll be great if we can make all the tests relatively consistent

Reply:
Yes you are right, I'd like to improve/refactor the entire foundry test base to standardize everything and make it consistent.

But we'll have to wait until everything has been migrated from hardhat to foundry I think.

wdyt ?